kubevela.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: kubevela
+  version: 1.6.7
+  epoch: 0
+  description: KubeVela is a modern application delivery platform that makes deploying and operating applications across today's hybrid, multi-cloud environments easier, faster and more reliable
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - build-base
+      - git
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/kubevela/kubevela/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 0b4d43518692e5c758148d1f8d78fe98f3134082cb4b3b49ca1200e3e44e8ade
+  - runs: |
```

Comment:
I found two targets in the makefile of the kubevela, which abstracts all the details for us and makes this process easy to read, WDYT? It seems that the only thing that we need to do is run the "make vela-cli and make manager" and then install the binaries to the target dir of melange:

➡️https://github.com/kubevela/kubevela/blob/master/Makefile#L25
➡️https://github.com/kubevela/kubevela/blob/master/makefiles/build.mk#L3

Reply:
Thanks, have already noticed. But some packages seem used non-make way (even though projects provide a Makefile), so I'm curious if it's intentional. I can change to `make` anyway. 🤔